src/monitor/pgautofailover.sql

```diff
@@ -345,6 +348,33 @@ $$;
 grant execute on function pgautofailover.get_coordinator(text)
    to autoctl_node;
 
+
+CREATE FUNCTION pgautofailover.get_most_advanced_standby
```

Comment:
maybe `get_most_advanced_standby ` ->  `get_most_advanced_standby_reported_lsn`

Reply:
I think we have long enough names as it is, and that the context is clear enough. The only LSN we have on the monitor is the reportedLSN, so what else would we return in that function?